### ISSUE TITLE
Information in No list item result is not clearly visible after changing the orientation from landscape to portrait mode.

### ISSUE THREAD

**Issue by Yash14j** [microsoft/vscode#127393]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.4;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;#A11yMAS;#FTP;

### Environment Details:
Application Name: Bridge to Kubernetes (Mindaro)
Windows Version: Windows10

### Repro Steps:

1. Setup The Kubernetes extension.
2. Open visual studio code and load "Bikes" project.
3. Tab to "View" menu item in top navigation bar, press enter to expand and use down arrow key to Select "Command Pallet" option.
4. Search and select "Local Process with Kubernetes: Configure Local Process with Kubernetes”
5. Connect to Kubernetes panel will open.
6. Tab to bike options and press enter
7. Input “3000” as your local port and press enter.
8. Tab to Launch via NPM option and press enter to select it.
9. Change the orientation from landscape and portrait mode
10. Observe that the content in no result is not visible after changing the orientation from landscape to portrait mode.

### Actual:
Information in No list item result is not clearly visible after changing the orientation from landscape to portrait mode.

**Similar issue is also observed with Kubernetes Bikes accessibility Testing cluster**.

### Expected:
Information in No list item result should be completely visible after changing the orientation from landscape to portrait mode..
### User Impact:
Low vision users will not be able to view the Information of No list item

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!

Apart from that:

Please follow these recommendations when filling issues for VS Code:
 
* First try to find duplicates. Before filling any issue be aware of all the [issues tagged with accessibility](https://github.com/microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility). It is not a long list, please read through it in detail!
* Always test on VS Code insiders
* Attach gifs inline in the issue. Do not give a link to a 20 MB ppt video
* Do not write User Impact section, it is not useful
* Be responsive, if an issue requires more information please respond in a timely manner
 
Here’s an example of a good concise user filled issue https://github.com/microsoft/vscode/issues/96987
Here’s an example of a not a concise issue with too many details https://github.com/microsoft/vscode/issues/62033
 
The moto should be quality not quantity.

Thanks
